### PR TITLE
Field box heading

### PR DESCRIPTION
**Pull request by mrcodefinger**
Hi!
In the backend on the configuration page for the detailed property view, the headlines for the boxes in which the fields for the contact person and property view are dragged in were both "Fields".  This was sometimes a bit confusing when you need a lot of fields. I have specified this in this Pull Request. In time I would like to adapt it for the list view as well.
Kind regards!

### CODE REVIEW

**Comment by mrcodefinger** [comment]
I ask you to ignore the changes in composer.json, they are erroneously still included in this branch.

File: languages/onoffice-de_DE.po [hunk 1]
```diff
@@ -343,12 +343,18 @@ msgstr ""
...
 #: plugin/Gui/AdminPageSettingsBase.php:177
 msgid "Fields"
 msgstr "Felder"
 
+#: plugin/Gui/AdminPageEstateDetail.php:161
+msgid "Contact person fields"
+msgstr "Felder Ansprechpartner"
+
+#: plugin/Gui/AdminPageEstateDetail.php:176
+msgid "Real estate data fields"
```
Comment (jayay) [inline languages/onoffice-de_DE.po:355]: The wording is different between "Contact person fields" and "Real estate data fields". I would prefer if the real estate variant would be "Real estate fields". Also, if this is a heading, I would prefer if the first character of every word was uppercase for the English language.

For the German ones: Wouldn't "Ansprechpartner-Felder" and "Immobilien-Felder" sound better?

File: languages/onoffice-es_ES.po [hunk 2]
```diff
@@ -236,10 +236,18 @@ msgstr "Editar página"
...
+#: plugin/Gui/AdminPageSettingsBase.php:181
 msgid "Fields"
 msgstr "Campos"
 
+#: plugin/Gui/AdminPageEstateDetail.php:161
+msgid "Contact person fields"
+msgstr "Campos de la persona de contacto"
+
+#: plugin/Gui/AdminPageEstateDetail.php:176
+msgid "Real estate data fields"
```
Comment (jayay) [inline languages/onoffice-es_ES.po:248]: See above

File: languages/onoffice-it_IT.po [hunk 3]
```diff
@@ -234,10 +234,18 @@ msgstr "Modifica pagina"
...
+#: plugin/Gui/AdminPageSettingsBase.php:181
 msgid "Fields"
 msgstr "I campi"
 
+#: plugin/Gui/AdminPageEstateDetail.php:161
+msgid "Contact person fields"
+msgstr "Campi della persona di contatto"
+
+#: plugin/Gui/AdminPageEstateDetail.php:176
+msgid "Real estate data fields"
```
Comment (jayay) [inline languages/onoffice-it_IT.po:246]: See above

File: languages/onoffice-nl_NL.po [hunk 4]
```diff
@@ -235,9 +235,17 @@ msgstr "Pagina bewerken"
...
 msgid "Fields"
-msgstr "Velden"
+msgstr "Velden contactpersoon"
+
+#: plugin/Gui/AdminPageEstateDetail.php:161
+msgid "Contact person fields"
+msgstr "Felder Ansprechpartner"
+
+#: plugin/Gui/AdminPageEstateDetail.php:176
+msgid "Real estate data fields"
```
Comment (jayay) [inline languages/onoffice-nl_NL.po:247]: See above

File: plugin/Gui/AdminPageEstateDetail.php [hunk 5]
```diff
@@ -173,7 +173,7 @@ public function renderContent()
 		echo '</div>';
 		echo '<div class="fieldsSortable postbox" id="'
 			.esc_attr(self::getSpecialDivId(onOfficeSDK::MODULE_ESTATE)).'">';
-		echo '<h2 class="hndle ui-sortable-handle"><span>'.__('Fields', 'onoffice').'</span></h2>';
+		echo '<h2 class="hndle ui-sortable-handle"><span>'.__('Real estate data fields', 'onoffice').'</span></h2>';
```
Comment (jayay) [inline plugin/Gui/AdminPageEstateDetail.php:176]: See above

**Comment by jayay** [comment]
Is everything OK with this? Is this still relevant? Would you prefer if I took over?

**Comment by mrcodefinger** [comment]
It's okay, I just haven't gotten around to...

**Comment by mrcodefinger** [comment]
I have adapted the wording now, as you suggested.
Kind regards !

**Pull request merged**